### ISSUE TITLE
Registry not assigning system:authenticated role when "anon" project created

### ISSUE THREAD

**Issue by aweiteka** [cockpit-project/cockpit#6711]
Cockpit version 140
Origin version v1.5.0

When creating new project with "Anonymous" access via the registry console the `system:authenticated` role is not assigned. This means that anonymous users could `docker pull` an image but *authenticated* users (the only kind in the console) cannot view anonymous projects in the console.
 
**Output after UI "New project"**
```
$ oc get rolebindings -n authtest
NAME                    ROLE                    USERS      GROUPS                            SERVICE ACCOUNTS   SUBJECTS
admin                   /admin                  aweiteka
registry-admin          /registry-admin         aweiteka
registry-viewer         /registry-viewer                   system:unauthenticated
system:deployers        /system:deployer                                                     deployer
system:image-builders   /system:image-builder                                                builder
system:image-pullers    /system:image-puller               system:serviceaccounts:authtest
```

**Workaround**

Add group system:authenticated to registry-viewer role for the project

```
$ oc policy add-role-to-group registry-viewer system:authenticated
$ oc get rolebindings -n authtest
NAME                    ROLE                    USERS      GROUPS                                         SERVICE ACCOUNTS   SUBJECTS
admin                   /admin                  aweiteka
registry-admin          /registry-admin         aweiteka
registry-viewer         /registry-viewer                   system:unauthenticated, system:authenticated
system:deployers        /system:deployer                                                                  deployer
system:image-builders   /system:image-builder                                                             builder
system:image-pullers    /system:image-puller               system:serviceaccounts:authtest
```

**Comment by aweiteka** [comment]
cc @dmsimard

**Comment by petervo** [comment]
@aweiteka I fixed on our side in #6714, but it seems pretty unintutive to me that openshift behaves like that. I would have expected ```system:unauthenticated``` to implicitly grant ```system:authenticated```. Worth an issue upstream?

**Comment by aweiteka** [comment]
> it seems pretty unintutive to me that openshift behaves like that. I would have expected system:unauthenticated to implicitly grant system:authenticated.

I agree. And it's interesting this would be a regression since anonymous access from the console has been working for a long time.

> Worth an issue upstream?

https://github.com/openshift/origin/issues/14381
